### ISSUE TITLE
 Example 8.13: Canonical YAML seem to be incorrect

### ISSUE THREAD

**Issue by dervus** [yaml/yaml-spec#24]
``` yaml
>
 folded
 line

 next
 line
   * bullet

   * list
   * line

 last
 line↓
↓
# Comment
```

``` yaml
%YAML 1.2

---
!!str "\n\
      folded line\n\
      next line\n\
      \  * bullet\n
      \n\
      \  * list\n\
      \  * lines\n\
      \n\
      last line\n"
```

Leading LF shouldn't be there.

**Comment by eemeli** [comment]
The typo here is in the left-hand YAML. After Example 8.10 there's a mention "The following three examples duplicate this example", and all four of these duplicates except for 8.13 start the block scalar with an empty line.

**Comment by ingydotnet** [comment]
Fixed in 610043caaabc2b40a0fc1fc5977ed9296f1eaab7
